java/core/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.lancedb</groupId>
+        <artifactId>lancedb-parent</artifactId>
+        <version>0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>lancedb-core</artifactId>
+    <name>LanceDB Core</name>
+    <packaging>jar</packaging>
```

Comment:
what's the relationship between this and the lancedb/lance/pull/2115?

AFAIK these cannot be identical. (maybe this should be lancedb-core and the other one should be lance-core?)

Reply:
@walterddr This is the LanceDB java module with LanceDB functionalities e.g. embeddings, table names
The other one is Lance java module.

Put it to the sub module to give space if we want to put other connectors e.g. Spark connectors directly in LanceDB java side

Reply:
ok i see. so lancedb-core and lance-core are both core modules and doesn't depend on each other (and thus are independent basically?) nice i will pull in both in the trino side.

Reply:
Yeah yeah!! From the Java Side you need to import both
The only problem later is the packaged jar may be bigger, but we can consider that issue later